driver/src/main/java/org/neo4j/driver/v1/Values.java

```diff
@@ -27,6 +27,7 @@
 import java.util.Map;
 
 import org.neo4j.driver.v1.exceptions.ClientException;
+import org.neo4j.driver.v1.internal.util.Extract;
```

Comment:
I don't see how we would do that in a convenient way for a class like `Values` that has static methods. This surely contains implementation code. For all non-utility classes I'm happy to try separate between interface and implementation as much as possible.